src/normalise-svg.js

```diff
@@ -3,33 +3,33 @@
  * and full commands, rather than relative coordinates
  * and/or shortcut commands.
  */
-function normalizePath(d) {
+export function normalisePath (d) {
```

Comment:
This is a singleton file, exposing a single function: this can be a default export.
```suggestion
export default function normalisePath (d) {
```